common/views/components/ImageViewer/ImageViewer.js

```diff
@@ -183,9 +182,10 @@ const ImageViewer = ({ id, contentUrl, infoUrl, width }: ImageViewerProps) => {
           viewerVisible={showViewer}
           id={id}
           handleViewerDisplay={handleViewerDisplay}
+          infoUrl={infoUrl}
         />
       )}
-    </ImageInfoContext.Provider>
+    </Fragment>
```

Comment:
@wellcometrust/experience-devs what do we think, `<Fragment>` or `<>`?

Reply:
We wouldn't have to do any importing if we went with `<>`, so maybe that? Should be a fairly straightforwards find/replace for what we have already?